release-notes/README.md

```diff
@@ -26,6 +24,18 @@ This fixes a regression found in 1.0.13 due to a change in the underlying plafor
 **Release Date:** August 14, 2019  
 **Documentation:** [Release Notes 2.1.0](ugandaemr-210.md)
 
+##3.0.0
+
 **Release Date:** August 14, 2019  
 **Documentation:** [Release Notes 3.0.0](ugandaemr-300.md)
 
+## 3.0.4
+
+**Release Date:** May 14, 2020  
+**Documentation:** [Release Notes 3.0.4](ugandaemr-304.md)
+
+##3.1.3
```

Comment:
@ssmusoke please make this  a heading, currently its not.

Reply:
The heading is 2 hash signs, see 3.0.0 if you are worried about spaces they do not matter